src/plugins/unified_search/server/autocomplete/autocomplete_service.ts

```diff
@@ -32,6 +32,7 @@ export class AutocompleteService implements Plugin<void> {
         terminateAfter: moment.duration(terminateAfter).asMilliseconds(),
         timeout: moment.duration(timeout).asMilliseconds(),
       }),
+      getInitializerContextConfig: () => this.initializerContext.config,
```

Comment:
Q. Should the value returned here be Immutable? or `Object.freeze` so that no chagnes to it are allowed by whom ever consumes it?

Reply:
https://github.com/elastic/kibana/pull/145883/commits/6fac15aaf52f158bdd58b69d6226ef772d61ac87

Reply:
Using clone instead: https://github.com/elastic/kibana/pull/145883/commits/0256ccd199671578752456ef328e50e51eae3085